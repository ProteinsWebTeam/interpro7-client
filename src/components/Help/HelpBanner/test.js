import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

import { HelpBanner } from '.';

// TODO: the shallow render  is having issues with React.memo and
// this needs to be changed to TestRenderer, but that approach has
// issues with connects in the inner tree.
const renderer = new ShallowRenderer();

describe.skip('<HelpBanner />', () => {
  test('should render', () => {
    renderer.render(<HelpBanner />);
    expect(renderer.getRenderOutput()).toMatchSnapshot();
  });
});
