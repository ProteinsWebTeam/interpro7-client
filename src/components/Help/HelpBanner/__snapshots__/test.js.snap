// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HelpBanner /> should render 1`] = `
<div
  className="help-banner flex-card"
>
  <header>
    <span
      className="icon icon-common font-l"
      data-icon=""
    />
     
    Need more Help?
  </header>
  <div>
    If you need more info on 
    , you can either look at the:
    <br />
    <span
      className="icon icon-common font-l"
      data-icon=""
    />
     
    <Memo(Connect(Link))
      className=""
      to={
        Object {
          "description": Object {
            "other": Array [
              "help",
              "documentation",
            ],
          },
        }
      }
    >
      Documentation page
    </Memo(Connect(Link))>
    <br />
    or
     
    <Memo(Connect(Link))
      href="http://www.ebi.ac.uk/support/interpro-general-query"
      target="_blank"
    >
      contact us
    </Memo(Connect(Link))>
     
    directly with your question.
  </div>
</div>
`;
